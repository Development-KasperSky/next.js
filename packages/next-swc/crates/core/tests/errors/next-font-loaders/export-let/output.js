import firaCode from '@next/font/google/target.css?{"arguments":[],"import":"Abel","path":"pages/test.tsx"}';
import inter from '@next/font/google/target.css?{"arguments":[],"import":"Inter","path":"pages/test.tsx"}';
import React from 'react';
export { firaCode };
export { inter };
