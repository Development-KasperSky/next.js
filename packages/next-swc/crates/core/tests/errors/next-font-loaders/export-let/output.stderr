
  x Font loader calls must be assigned to a const
   ,-[input.js:4:1]
 4 | export let firaCode = Abel()
   :        ^^^^^^^^^^^^^^^^^^^^^
   `----

  x Font loader calls must be assigned to a const
   ,-[input.js:5:1]
 5 | export var inter = Inter()
   :        ^^^^^^^^^^^^^^^^^^^
   `----
